cmd/util.go

```diff
@@ -27,14 +27,16 @@ var (
 
 var psFunc func(string) string
 var statFunc func(string) (os.FileInfo, error)
-var getBinariesFunc func(*viper.Viper) (map[string]string, error)
+var getBinariesFunc func(*viper.Viper, check.NodeType) (map[string]string, error)
 var TypeMap = map[string][]string{
 	"ca":         []string{"cafile", "defaultcafile"},
 	"kubeconfig": []string{"kubeconfig", "defaultkubeconfig"},
 	"service":    []string{"svc", "defaultsvc"},
 	"config":     []string{"confs", "defaultconf"},
 }
 
+var errMissingKubectlKubelet = fmt.Errorf(`unable to find the programs kubectl or kubelet in the PATH`)
```

Comment:
It would be clearer to have this as an error inline in the code, it doesn't need to be a variable